README.md

```diff
@@ -47,6 +47,25 @@ The simplest usage is just to invoke the robot:
 
     $ time_log_robot
 
+### Configuration
```

Comment:
Good idea on the rearrange. I'd suggest that the chunk of text lines 102-115 ("On start, the robot will ask you for these details..." through "...so open up that file and edit to your heart's content") should go here.